controllers/managedocs_controller.go

```diff
@@ -1461,17 +1463,46 @@ func (r *ManagedOCSReconciler) reconcileRookCephOperatorConfig() error {
 }
 
 func (r *ManagedOCSReconciler) reconcileEgressNetworkPolicy() error {
-	// Fix for non-converged deployment type will be provided in upcoming PR
-	if r.DeploymentType != convergedDeploymentType {
-		r.Log.Info("Non converged deployment, skipping reconcile for egress network policy")
-		return nil
-	}
 	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.egressNetworkPolicy, func() error {
 		if err := r.own(r.egressNetworkPolicy); err != nil {
 			return err
 		}
 		desired := templates.EgressNetworkPolicyTemplate.DeepCopy()
 
+		if r.DeploymentType != convergedDeploymentType {
```

Comment:
Do we need the new code on both provider and consumer?

Reply:
yes, both modes should set the network policy.

Reply:
Ok